README.md

```diff
@@ -44,6 +45,36 @@ To run the tests
     make test
 
 
+.gitAttributes
+--------------
+
+Like in linguist you can override the strategies via `.gitattributes` file.
+Add a `.gitattributes` file to the directory and use the enry matchers `enry-documentation`,`enry-language` or `enry-vendored` to do the override.
```

Comment:
Yes, but if you name them differently you're forcing the user to have to support both linguist and enry separately. If you just use the same name, enry will already support all those attributes already customized by users for linguist.